sdk/src/utils/mime.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2022 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License,
+// Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+// or the MIT license (http://opensource.org/licenses/MIT),
+// at your option.
+
+// Unless required by applicable law or agreed to in writing,
+// this software is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR REPRESENTATIONS OF ANY KIND, either express or
+// implied. See the LICENSE-MIT and LICENSE-APACHE files for the
+// specific language governing permissions and limitations under
+// each license.
+
+pub fn extension_to_mime(extension: &str) -> Option<&'static str> {
```

Comment:
Maybe some docs here since this is a new public interface

Reply:
I added docs but the the utils stuff is all crate private so it won't be in the public SDK, unless we want to add it there.

Reply:
ah, got it!